docs/intro/install.rst

```diff
@@ -118,6 +118,27 @@ Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
 
   conda install -c conda-forge scrapy
 
+To install Scrapy on Windows using ``pip`` :-
+
+1. Install latest version of `Python`_ (Recommended to tick [Add Python to PATH] checkbox during installation)
+
+2. Install or upgrade pip::
+
+    python -m pip install --upgrade pip
+
+3. Create Venv (Virtual Environment)::
+
+    python -m venv [env name]
+
+4. Download `Microsoft Visual C++ 14.0`_
```

Comment:
- I would recommend using `#. ` instead of `4. `, so that we do not need to worry about list numbers when adding, removing or moving steps in the future.

- Do we need to indicate a specific version?

- The link at the moment calls the package “Microsoft C++ Build Tools”. If that’s the name, we should use it here. If users are meant to install “Microsoft Visual C++” using the “Microsoft C++ Build Tools” installer, we should explain that (e.g. “install the Microsoft C++ Build Tools, which include the Microsoft Visual C++ required to install some Scrapy dependencies”).